src/assets/TestFramework/DeploymentProperties.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Resources.Models
```

Comment:
Is this namespace correct?

Reply:
renamed to Azure.ResourceManager.Resources.Models